nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core.Test/Files/GlobalJsonBuildFileTests.cs

```diff
@@ -55,6 +53,7 @@ public void GlobalJson_GetDependencies_ReturnsDependencies()
     {
         var expectedDependencies = new List<Dependency>
         {
+            new("Microsoft.NET.Sdk", "6.0.405", DependencyType.MSBuildSdk),
```

Comment:
We report .NET SDK version info now but remove it on the Ruby side, since it has not been supported by dependabot.